iso_config.yaml

```diff
@@ -0,0 +1,37 @@
+log_file: "/var/tmp/host-os-iso-creator.log"
+mock_config: "./extras/centOS/7.2/mock/spin-iso-epel-7-ppc64le.cfg"
+rpm_dir: "./results"
```

Comment:
This could be a common config between this and the host-os-build "result_dir" RPM. It would be easier to generate the RPMs and then create the ISO in sequence. Maybe we could merge this file into config.yaml and use a sub-node "mock_iso" to separate the specific configs?